python/tvm/runtime/module.py

```diff
@@ -370,6 +390,44 @@ def export_library(self, file_name, fcompile=None, addons=None, workspace_dir=No
 
         return fcompile(file_name, files, **kwargs)
 
+    def export_model_library_format(self, codegen_dir: str):
```

Comment:
can we change it to micro.export_model_library_format?

Reply:
May I ask why though? I can't immediately see why should it be restricted to micro. This format should be beneficial to flows that would otherwise use module.save() traversing the import tree.

Reply:
@manupa-arm I have a forthcoming RFC which makes some complementary changes to the codegen pipeline that then allow for a more rigorous discussion about this format. after that RFC has been discussed, we can then consider promoting Model Library Format to `tvm.runtime.Module` in a way that's accommodating to more users.